analyzer/tools/build-logger/Makefile.manual

```diff
@@ -113,6 +113,3 @@ ldlogger_32.so: $(LDLOGGER_LIB_SOURCES) $(LDLOGGER_LIB_HEADERS)
 # ldlogger lib 64
```

Comment:
Seems like `Makefile.manual` is not needed anymore. Could we just remove it?

Reply:
Sure, now I removed all references to it. Also deleted the file.